docker-compose.yml

```diff
@@ -1,21 +1,34 @@
 version: "3.4"
 services:
-  lodestar:
-    build: .
+  beacon_node:
+    build: ../lodestar
+    image: lodestar/lodestar
     restart: always
     volumes:
-      - lodestar:/root/.local/share/lodestar
-    command: beacon
+      - beacon_node:/root/.local/share/lodestar
+      - /root/.local/share/lodestar/${NETWORK:-pyrmont}:/root/.local/share/lodestar/${NETWORK:-pyrmont}
+    command: beacon --network ${NETWORK:-pyrmont} --api.rest.host 0.0.0.0
     ports:
       - "9000:9000"
 
+  validator:
+    image: lodestar/lodestar
+    restart: always
+    volumes:
+      - validator:/root/.local/share/validator
+      # NOTE: mapping of ${NETWORK} volume assumes that a validator has been manually imported or created here
+      - /root/.local/share/lodestar/${NETWORK:-pyrmont}:/root/.local/share/lodestar/${NETWORK:-pyrmont}
```

Comment:
Assuming `/root/.local/share/lodestar` exists in all hosts that may run this command will cause issues. Especially when running it in non-VPS setups. I would suggest to use a relative path here with some validator path flags